### PR TITLE
varLib CFF fixes

### PR DESCRIPTION
**Pull request by miguelsousa**

### CODE REVIEW

File: Lib/fontTools/varLib/cff.py [hunk 1]
```diff
@@ -205,11 +205,10 @@ def merge_PrivateDicts(top_dicts, vsindex_dict, var_model, fd_map):
 				try:
 					values = [pd.rawDict[key] for pd in pds]
 				except KeyError:
-					del private_dict.rawDict[key]
```
Comment (anthrotype) [inline Lib/fontTools/varLib/cff.py]: are you sure you want to delete this line?

**Comment by anthrotype** [comment]
I'm not sure I fully understand the issue here (especially the last commit 19f7e31), so i'd like to ask @readroberts to take a look instead.

File: Lib/fontTools/varLib/cff.py (hunk 1 above)
Comment (miguelsousa) [reply Lib/fontTools/varLib/cff.py]: Yes. The VF will have the item with the values of the default master. No blends. I think that's better than dropping it entirely.

**Comment by miguelsousa** [comment]
@anthrotype any chance that a new FontTools version be released this week? We need to integrate these fixes in the AFDKO.

**Comment by anthrotype** [comment]
I'm busy attending the TDi programme in Reading, but I'll try to find the time later this week.

**Comment by anthrotype** [comment]
https://github.com/fonttools/fonttools/releases/tag/3.43.2

**Comment by miguelsousa** [comment]
@anthrotype thanks

**Pull request merged**